static/app/views/performance/newTraceDetails/traceContextVitals.tsx

```diff
@@ -199,19 +221,110 @@ const VitalPillValue = styled('div')`
   font-size: ${p => p.theme.fontSizeLarge};
 `;
 
-const VitalMetersContainer = styled('div')`
+const VitalMetersContainer = styled('div')<{hasTraceTabsUi?: boolean}>`
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
+  align-items: baseline;
   gap: ${space(1)};
-  width: 100%;
+  width: ${p => (p.hasTraceTabsUi ? 'auto' : '100%')};
 `;
 
-const NoValueContainer = styled('span')`
+const SecondaryVitalsCount = styled('span')`
   color: ${p => p.theme.subText};
-  font-size: ${p => p.theme.headerFontSize};
+  font-size: ${p => p.theme.fontSizeSmall};
 `;
 
-function NoValue() {
-  return <NoValueContainer>{' \u2014 '}</NoValueContainer>;
+const SecondaryVitalsCountContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  white-space: nowrap;
+  gap: ${space(0.5)};
+  text-align: left;
+`;
+
+function getPrimaryVitalsCount(
```

Comment:
Only new functions are: getPrimaryVitalsCount, getVitalInfo getVitalDetails. Rest are just moved to the bottom.